### PR TITLE
Example

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
@fbudin69500 When built against an ITK that includes [this modules examples patch](http://review.source.kitware.com/#/c/23022/) could you please investigate and fix why this error occurs?

```
[2/11] Linking CXX executable bin/PhaseCorrelationImageRegistration
FAILED: bin/PhaseCorrelationImageRegistration
: && /home/matt/.config/dotfiles/bin/ccache-clang++  -Os -DNDEBUG   examples/CMakeFiles/PhaseCorrelationImageRegistration.dir/PhaseCorrelationImageRegistration.cxx.o  -o bin/PhaseCorrelationImageRegistration  -Wl,-rpath,/home/matt/bin/ITK-Wrap-MinSizeRel/lib:/usr/lib/x86_64-linux-gnu/hdf5/serial /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOBMP-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOBioRad-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOBruker-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOGDCM-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkgdcmMSFF-4.13.so.1 /usr/lib/x86_64-linux-gnu/libexpat.so /usr/lib/x86_64-linux-gnu/libz.so /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOGE-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOIPL-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOGIPL-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOHDF5-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOJPEG-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOLSM-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOMINC-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOMRC-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOMeta-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKMetaIO-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIONIFTI-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIONRRD-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOPNG-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOStimulate-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOTIFF-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOVTK-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKStatistics-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKTransform-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkNetlibSlatec-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkgdcmDICT-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkgdcmIOD-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkgdcmDSED-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkgdcmCommon-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKIOImageBase-4.13.so.1 /usr/lib/x86_64-linux-gnu/libz.so /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKCommon-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitksys-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libITKVNLInstantiation-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkvnl_algo-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkvnl-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkv3p_netlib-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitknetlib-4.13.so.1 /home/matt/bin/ITK-Wrap-MinSizeRel/lib/libitkvcl-4.13.so.1 -lm -lpthread -lm -ldl -Wl,-rpath-link,/home/matt/bin/ITK-Wrap-MinSizeRel/lib:/usr/lib/x86_64-linux-gnu/hdf5/serial && :
examples/CMakeFiles/PhaseCorrelationImageRegistration.dir/PhaseCorrelationImageRegistration.cxx.o: In function `main':
/home/matt/src/ITKMontage/examples/PhaseCorrelationImageRegistration.cxx:(.text+0x39e): undefined reference to `itk::TransformFileWriterTemplate<double>::New()'
/home/matt/src/ITKMontage/examples/PhaseCorrelationImageRegistration.cxx:(.text+0x3c7): undefined reference to `itk::TransformFileWriterTemplate<double>::SetInput(itk::Object const*)'
/home/matt/src/ITKMontage/examples/PhaseCorrelationImageRegistration.cxx:(.text+0x3e9): undefined reference to `itk::TransformFileWriterTemplate<double>::Update()'
examples/CMakeFiles/PhaseCorrelationImageRegistration.dir/PhaseCorrelationImageRegistration.cxx.o:(.data+0xa0): undefined reference to `itk::HDF5TransformIOFactoryRegister__Private()'
examples/CMakeFiles/PhaseCorrelationImageRegistration.dir/PhaseCorrelationImageRegistration.cxx.o:(.data+0xa8): undefined reference to `itk::MatlabTransformIOFactoryRegister__Private()'
examples/CMakeFiles/PhaseCorrelationImageRegistration.dir/PhaseCorrelationImageRegistration.cxx.o:(.data+0xb0): undefined reference to `itk::TxtTransformIOFactoryRegister__Private()'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.9.5)
+project(ITKMontageExamples)
+
+find_package(ITK REQUIRED
+  COMPONENTS ITKImageIO ITKTransformIO Montage
+  )
+include(${ITK_USE_FILE})
+find_package(ITK REQUIRED
```
Comment (dzenanz) [inline examples/CMakeLists.txt]: Again find_package?

Comment (thewtex) [reply examples/CMakeLists.txt]: Yes, I did not mean to commit this -- good catch!

Comment (thewtex) [reply examples/CMakeLists.txt]: @fbudin69500 and this is the cause of the "issue" I reported above -- nevermind! :put_litter_in_its_place:

**Pull request merged**